lib/show_n_tell_web/controllers/auth_controller.ex

```diff
@@ -11,6 +11,18 @@ defmodule ShowNTellWeb.AuthController do
     |> render_result(conn)
   end
 
+  def show(conn, _params) do
+    [authorization_header] =
+      conn
+      |> get_req_header("authorization")
+
+    "bearer " <> token = authorization_header
```

Comment:
Had to lookup `<>` operator again. In Ruby it is recommended to use string interpolation for performance reasons:

`auth_header = "bearer #{token}"`

In Elixir - I'm not sure.

Reply:
@alex-kovshovik I am using `<>` for pattern matching. I needed to extract only the token from the string "bearer 232rqf34fq3f34"

Reply:
I don't think you can pattern match with string interpolation: 
```iex(2)> "header #{token}" = header
** (MatchError) no match of right hand side value: "bearer asdfa2423f2"```

Reply:
Ah, I see, I didn't get it then :) I forgot you could do that in Elixir - this is awesome!